uploader/uploader.go

```diff
@@ -43,7 +43,7 @@ func (self *Uploader) Upload(buf *bytes.Buffer, f storage.Image) (string, error)
 	t := log.Start()
 	defer log.End(t)
 
-	object := &gcs.Object{Name: f.Filename}
+	object := &gcs.Object{Name: f.Filename, CacheControl: "max-age=15552000"}
```

Comment:
Calculate seconds as constant value at the start of this file.
It may be as well to explain in code which can be expressed in code.

Example:

```go
const SIX_MONTHS = 60 * 60 * 24 * 30 * 6
```

And you can use it:

```go
	object := &gcs.Object{Name: f.Filename, CacheControl: fmt.Sprintf("max-age=%d", SIX_MONTHS)}
```